Galactic Repository Missing Items

Hi all,

I'm new to Github and ROS, but trying to get everything installed and it looks like there are some things missing from the Galactic install. First, on the main page it says Galactic Binary is passing, but when you go to the tree it says it's failing.(https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/tree/galactic)

Then when I clone the files, I don't have a galactic version of the "-not-released" repos. So I delete that out of the line.

Then when I run the install line (colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release) I get these errors:
1) --- stderr: ur_controllers  
2) [file] error: ‘CallbackReturn’ in namespace ‘controller_interface’ does not name a type
   65 |   controller_interface::CallbackReturn on_configure(const rclcpp_lifecycle::State& previous_state) override;
[other file] error: ‘CallbackReturn’ in namespace ‘controller_interface’ does not name a type
   67 |   controller_interface::CallbackReturn on_activate(const rclcpp_lifecycle::State& previous_state) override;
69,71,61,86,107,113,44,154,238,283,57,55
3) Ending with this: make[2]: *** [CMakeFiles/ur_controllers.dir/build.make:76: CMakeFiles/ur_controllers.dir/src/speed_scaling_state_broadcaster.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/ur_controllers.dir/build.make:63: CMakeFiles/ur_controllers.dir/src/scaled_joint_trajectory_controller.cpp.o] Error 1
make[2]: *** [CMakeFiles/ur_controllers.dir/build.make:89: CMakeFiles/ur_controllers.dir/src/gpio_controller.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:78: CMakeFiles/ur_controllers.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
---
Failed   <<< ur_controllers [8.26s, exited with code 2]

Summary: 27 packages finished [14.9s]
  1 package failed: ur_controllers
  1 package had stderr output: ur_controllers
  2 packages not processed


I'm going to try and get the Foxy version running, but if someone could help me out that would be greatly appreciated!

--- a/README.md
+++ b/README.md
@@ -173,7 +173,7 @@ binary package installation unless you want to join development and submit chang
 4. Pull relevant packages, install dependencies, compile, and source the workspace by using:
    ```
    cd $COLCON_WS
-   git clone https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver.git src/Universal_Robots_ROS2_Driver
+   git clone -b galactic https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver.git src/Universal_Robots_ROS2_Driver
    rosdep update
    rosdep install --ignore-src --from-paths src -y
    colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release

--- a/README.md
+++ b/README.md
@@ -159,9 +159,9 @@ binary package installation unless you want to join development and submit chang
    the main branch. Therefore you might not find a dedicated branch for a recent ROS2 version. In
    that case, please use the `main` branch.
 
-2. Make sure that `colcon`, its extensions and `vcs` are installed:
+2. Make sure that `colcon` and its extensions are installed:
    ```
-   sudo apt install python3-colcon-common-extensions python3-vcstool
+   sudo apt install python3-colcon-common-extensions
    ```
 
 3. Create a new ROS2 workspace:
@@ -174,9 +174,8 @@ binary package installation unless you want to join development and submit chang
    ```
    cd $COLCON_WS
    git clone https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver.git src/Universal_Robots_ROS2_Driver
-   vcs import src --skip-existing --input src/Universal_Robots_ROS2_Driver/Universal_Robots_ROS2_Driver-not-released.${ROS_DISTRO}.repos
    rosdep update
-   rosdep install --ignore-src --from-paths src -y -r
+   rosdep install --ignore-src --from-paths src -y
    colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
    source install/setup.bash
    ```

